curvefs/src/client/rpcclient/metaserver_client.cpp

```diff
@@ -107,6 +107,13 @@ class MetaServerClientRpcDoneBase : public google::protobuf::Closure {
     MetaServerClientMetric *metric_;
 };
 
+void MetaServerClientImpl::SetCreateTime(Time* tm) {
```

Comment:
`SetCreateTime` should not be a member function, define it as a free function is ok.